src/App.vue

```diff
@@ -20,6 +23,12 @@ import ModuleNav from '@/components/ModuleNav'
 import Footer from '@/components/Footer'
 
 export default {
+  data () {
+    return {
+      loggedIn: false,
+      authenticated: false
```

Comment:
Noting for future reference that right now these are the lines we need to set to `true` in the dev tools in order to be logged in and see the feedback stuff